src/uthash.h

```diff
@@ -26,9 +26,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #define UTHASH_VERSION 2.0.1
 
+#ifndef uthash_noheaders
```

Comment:
Including stdint.h is not OK :) I don't know how I missed it, I'll take another look. As to the naming, I followed the names used for overriding function calls, like `uthash_xxx`. Is the rule here - functions - lower case names, others - upper case? I assumed hooks are lower case.